source/manual/world-taxonomy.html.md

```diff
@@ -0,0 +1,75 @@
+[live-latvia]: https://www.gov.uk/world/living-in-latvia
+[schema]: https://github.com/alphagov/govuk-content-schemas
+[passports-generic]: https://www.gov.uk/world/passports-and-emergency-travel-documents
+[taxon-list]: https://content-tagger.publishing.service.gov.uk/taxons/91b8ef20-74e7-4552-880c-50e6d73c2ff9
+[passports]: https://www.gov.uk/world/passports-and-emergency-travel-documents
+[vietnamese-passports]: https://www.gov.uk/world/passports-and-emergency-travel-documents-vietnam
+
+## Differences between topic taxonomy and world taxonomy
+
+Whilst world taxons are treated in the same way as the regular [taxonomy][taxonomy] on a data level, they are a separate entity and do not share the same root. They cannot be found by visiting the root of the GOV.UK taxonomy meaning that they are simultaneously part of and not part of the core taxonomy.
+
+The two have very different structures. The world taxon structure has a breadth of over 200, one for each country and a few generic. The majority of these taxons have only one level below them while the rest have none. The topic taxonomy is an entirely different shape with 21 taxons under the GOV.UK home page each of which with a variable depth up to 5 or 6 levels.
+
+GOV.UK has 3780 published taxons as of the time of writing (03/02/2020), of those 2488 are world taxons meaning that they represent a substantial majority of all taxons.
+
+Unlike the rest of the taxonomy only content associated with [specific organisations][spec-orgs] is allowed to be tagged to world taxons (DFID, FCO, UKVI, DIT, DExEU) and only [particular document types][specific-doc-types] (guidance, forms, detailed guides, and document collections) may be tagged to them. Presumably this is because the world pages are intended to be curation mechanisms for FCO and similar rather than content tagged to a topic.
+
+They are rendered in a completely different format to regular topic taxons. Compare https://www.gov.uk/education/running-and-managing-a-school with https://www.gov.uk/world/yemen.
+
+[taxonomy]: https://docs.publishing.service.gov.uk/manual/taxonomy.html
+[spec-orgs]: https://github.com/alphagov/whitehall/blob/56006c6f6ba033fbe450ef91d46204499e62e337/config/worldwide_tagging_organisations.yml
+[specific-doc-types]: https://github.com/alphagov/whitehall/blob/master/app/models/edition/taggable_organisations.rb#L4-L9
+
+## Problems with world taxonomy
+
+There have been a number of protracted discussions about whether this concept represents [technical debt for GOV.UK][tech-debt] tech community or if it is a form of product debt. Their existence is not well known among people who work on GOV.UK, despite it being a knowledge prerequisite to do any development work with the regular topic taxonomy. Any apps pulling taxons out of the Publishing API will get both so need to filter down to the ones they need. This means that all apps that use the taxonomy have to make concessions for the world taxonomy even if they don't use it.
```

Comment:
I don't think you need "There have been a number of protracted discussions about whether this concept represents [technical debt for GOV.UK][tech-debt] tech community or if it is a form of product debt." since that's kind of opinion.

I'd start with something like 

"The world taxonomy is not a well known part of GOV.UK and has a number of quirks that can make it problematic for developers working on GOV.UK. These quirks can produce a number of hard to diagnose problems."

Then I'd go with a first problem of:

"Any applications that either tag content to taxons or pull taxons out of the Publishing API have to consider them. Tagging to taxons requires an application to consider whether they will be accidentally overwriting any tagging to the world taxonomy. Pulling taxons from the Publishing API requires determining whether each item is part of the topic taxonomy or the world taxonomy."

Reply:
Nice, I've added these suggestions in :)